packages/backend/src/graphql/schema.graphql

```diff
@@ -37,6 +37,9 @@ type Query {
 
   "Fetch an Open Payments outgoing payment"
   outgoingPayment(id: String!): OutgoingPayment
+
+  "Fetch an Open Payments incoming payment"
+  incomingPayment(id: String!): IncomingPayment
```

Comment:
I wrote out the type definition for this in the original issue (and this isn't specific to this PR), but looking at the schema in general, I think going forward we should use `id: ID!` type for our id's. It doesn't _actually_ change anything, except for readability, but just makes the schema tidier IMO

Reply:
That makes sense